test/testSetupFile.js

```diff
@@ -1 +0,0 @@
-jest.setTimeout(10000);
```

Comment:
Older `jest-light-runner` versions did not support `setupFilesAfterEnv`, so this file was not loaded at all (and thus we don't really need it).

The latest `jest-light-runner` supports `setupFilesAfterEnv`, but it does not support `jest.setTimeout`.